packages/smooth_app/lib/generic_lib/widgets/smooth_text_form_field.dart

```diff
@@ -77,11 +82,15 @@ class _SmoothTextFormFieldState extends State<SmoothTextFormField> {
               }
             }
           },
+      onFieldSubmitted: widget.onFieldSubmitted,
+      style: TextStyle(fontSize: textSize),
+      cursorHeight: textSize * (textStyle.height ?? 1.4),
       decoration: InputDecoration(
+        //contentPadding: EdgeInsets.only(bottom: textSize),
```

Comment:
I think the comment is to be removed? Ryt?

Reply:
Ooops you're right 👍